src/test/java/cz/muni/fi/dao/TeamDaoTest.java

```diff
@@ -0,0 +1,87 @@
+package cz.muni.fi.dao;
+
+import cz.muni.fi.PersistenceApplicationContext;
+import cz.muni.fi.entities.Team;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.testng.AbstractTestNGSpringContextTests;
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+import java.util.List;
+
+/**
+ * @author Matus Macko
+ */
+@ContextConfiguration(classes = PersistenceApplicationContext.class)
+public class TeamDaoTest extends AbstractTestNGSpringContextTests {
+    @PersistenceContext
+    private EntityManager em;
+
+    @Autowired
+    private TeamDaoImpl teamManager = new TeamDaoImpl();
+
+    private Team team1;
```

Comment:
Maybe use better names for team and car like `Blue team`, Adamek don't like such a names